x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/common/http/BasicAuth.java

```diff
@@ -103,6 +98,20 @@ public static BasicAuth parse(XContentParser parser) throws IOException {
         return new BasicAuth(username, password);
     }
 
+    public static BasicAuth parse(XContentParser parser) throws IOException {
+        String type = null;
```

Comment:
This is not new, it was just yanked from `HttpAuthRegistry.java` since its still used in ser/deser of HttpRequest and some other classes. Id be happy to do it, but id like to make it a diff commit since this is just moving the code.